monitor/subscription.go

```diff
@@ -39,10 +39,28 @@ type NodeMonitor struct {
 	errHandlerCB     ErrHandler
 }
 
-// internal struct to manage various ids
-type itemIDs struct {
-	handle uint32 // client-provided
-	id     uint32 // from server
+// Item is a struct to manage Monitored Items
+type Item struct {
+	id     uint32     // from server
+	nodeID *ua.NodeID // from request
+	handle uint32     // client provided
+}
+
+// ID returns the MonitorItemID set by the server
+func (m Item) ID() uint32 {
+	return m.id
+}
+
+// NodeID returns the NodeID for the Item
+func (m Item) NodeID() ua.NodeID {
```

Comment:
any reason you return a copy of the node id?

Reply:
Fixed to return a pointer. Same question here as above regarding the use of pointers for small objects like this? Is it to make NodeID nullable?

Reply:
Same as above. No reason to make a copy of the node.